ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/MessageFrame.java

```diff
@@ -320,9 +320,7 @@ private MessageFrame(
     this.transactionHash = transactionHash;
     this.revertReason = revertReason;
 
-    this.warmedUpAddresses = new HashSet<>();
-    warmedUpAddresses.add(sender);
-    warmedUpAddresses.add(contract);
+    this.warmedUpAddresses = new HashSet<>(List.of(sender, contract));
```

Comment:
This winds up creating a new List object for each iteration, and creates a loop that can simply be unrolled to what is originally there.  Yes, it's two more lines of code but a lot less VM work.

Reply:
Sure thing, I'll remove it